.github/workflows/build.yml

```diff
@@ -50,8 +50,7 @@ jobs:
       - name: Package
         run: |
           cmake --build build --target install
-          cmake --build build --target appimage \
-            || (cat build/appimage.log && false)
```

Comment:
I guess you removed the `cat ...` because `try_quiet` already gives more precise info to stdout?

Reply:
Exactly, it's the same except no file is created